ang/dedupeSearchTasks.ang.php

```diff
@@ -0,0 +1,16 @@
+<?php
+// This file declares an Angular module which can be autoloaded
+// in CiviCRM. See also:
+// http://wiki.civicrm.org/confluence/display/CRMDOC/hook_civicrm_angularModules
```

Comment:
erm - maybe not - where is that boilerplate coming from?

Reply:
Civix

Reply:
@colemanw old version?